Dockerfile.discord~

```diff
@@ -0,0 +1,22 @@
+FROM python:3.7
```

Comment:
What's the difference between this file and `Dockerfile.discord`? Do we need both?